website/src/releases/2020-10/release-v17.mdx

```diff
@@ -0,0 +1,233 @@
+  },
+});
+```
+
+# Changes about the `mc-scripts` CLI
+
+## Removing deprecated options from `compile-html`
+
+Previously deprecated CLI options such as `--env`, `--csp`, `--headers`, have been removed.
+
+Additionally, the CLI flag `--use-local-assets` has been removed as well. The default behavior of `mc-scripts compile-html` now is to compile the assets locally, which is the only reasonable thing to do.
+
+When running the `mc-scripts compile-html` command, the `index.html` is compiled for production usage and it lives in the `public` folder, together with the other static assets. This is all you need to deploy your application.
+You can decide to [deploy the Custom Application statically to one of the popular cloud providers](/deployment/compiling-a-custom-application#deployment), or serve the files on your own using a static server.
+
+For example, to run locally the Custom Application using the production bundles:
+
+```console
+NODE_ENV=production MC_APP_ENV=development dotenv -- \
+  mc-scripts compile-html \
+  --transformer @commercetools-frontend/mc-dev-authentication/transformer-local.js
+
+mc-scripts serve
+```
+
+## Removing the `mc-http-server` package
+
+The `@commercetools-frontend/mc-http-server` package has been deprecated and won't be published anymore.
+
+With the usage of the `compile-html` command there is no need to have a pre-configured HTTP server anymore. If you are using this package, we recommend to use any other HTTP server package to serve your static files.
+
+<Info>
+
+Remember that after building your production bundles you need to [compile the Custom Application](/deployment/compiling-a-custom-application#deployment) for production usage.
+
+</Info>
+
+## New command `serve`
+
+We added a new command `mc-scripts serve` that can be used to start your Custom Application **locally** in `production` mode, after having it compiled.
+
+The command starts an HTTP server to serve the static assets from the `public` folder.
+
+<Warning>
+
+This command should only be used locally to test the Custom Application in production mode, as it contains the development routes for `/login` and `/logout`.
+
+</Warning>
+
+Before:
+
+```json title="package.json"
+{
+  "start:prod:local": "NODE_ENV=production MC_APP_ENV=development dotenv -- mc-http-server --use-local-assets"
+}
+```
+
+After:
+
+```json title="package.json"
+{
+  "compile-html:local": "NODE_ENV=production MC_APP_ENV=development dotenv -- mc-scripts compile-html --transformer @commercetools-frontend/mc-dev-authentication/transformer-local.js",
+  "start:prod:local": "yarn compile-html:local && mc-scripts serve",
+}
+```
+
+## Removing the `extract-intl` command
+
+The `mc-scripts extract-intl` command has been removed in favor of the [official `@formatjs/cli` package](https://formatjs.io/docs/tooling/cli#extraction).
+
+We recommend to update your script to extract Intl messages to use the `formatjs extract` command.
```

Comment:
```suggestion
We recommend to update your script to extract intl messages to use the `formatjs extract` command.
```

Reply:
In our docs, we always write it with capital letter, as it refers to the Intl object/feature.